api/sds.proto

```diff
@@ -141,9 +141,9 @@ message DownstreamTlsContext {
   // If specified, Envoy will reject connections without a valid and matching SNI.
   google.protobuf.BoolValue require_sni = 3;
 
-  oneof session_ticket_keys {
-    TlsSessionTicketKeys keys = 4;
-    SdsSecretConfig config = 5;
+  oneof session_ticket_keys_type {
+    TlsSessionTicketKeys session_ticket_keys = 4;
+    SdsSecretConfig session_ticket_keys_config = 5;
```

Comment:
I'd rather call this `sds_secret_config`, since it's definitely not a config for Session Ticket keys ;)

Reply:
I think the point (as I understand it) is that `session_ticket_keys_type` is lost from the generated code, right @ggreenway? So the goal is to make it a little more readable. Perhaps `session_ticket_sds_secret_config` ?

Reply:
@mattklein123 yes, that is my understanding of it.  @PiotrSikora if it is named sds_secret_config, it is bare in the DownstreamTlsContext with no indication that it is related to session ticket keys, as opposed to any other secret in that message.

Reply:
Right, sorry, I got confused with a separate message embedding `oneof`.

Nit: `session_ticket_keys_sds_secret_config` would be a bit more correct.